en/docs/test/open-api-console.md

```diff
@@ -1,3 +1,25 @@
-# OpenAPI Console
+Choreo allows you to test your APIs using the integrated OpenAPI Console, the generated cURL command, or by integrating Postman to Choreo. Once you have created an API and deployed it, it is ready to be tested. 
 
-A how-to guide on how to use OpenAPI console to test the component. 
\ No newline at end of file
+You can follow the steps below to test your REST API using the integrated OpenAPI Console:
+
+# Test your REST API
+
+!!! note
+    If you are on the deployment tab, click **Test**. in the **Development** card and continue from step 4 onwards. 
```

Comment:
Since you are making a specific reference to the tab, use bold and caps for "**Deployment** tab".

There is an unnecessary full stop after **Test**.

Reply:
done